docs/getting-started.md

```diff
@@ -18,33 +18,50 @@ Build all required artifacts and launch the
 dockerizied environment:
 
 ```bash
-    # Building and deploying kubevirt in Vagrant
-    export PROVIDER=k8s-1.9.3
-    make cluster-up
-    make cluster-sync
+# Build and deploy KubeVirt on Kubernetes 1.9.3 in our vms inside containers
+export KUBEVIRT_PROVIDER=k8s-1.9.3
+make cluster-up
+make cluster-sync
 ```
 
 This will create a VM called `node01` which acts as Kubernetes master and then
 deploy KubeVirt there. To create one or more nodes which will register
-themselves on master, you can use the `VAGRANT_NUM_NODES` environment variable.
+themselves on master, you can use the `KUBEVIRT_NUM_NODES` environment variable.
 This would create a master and one node:
 
 ```bash
-    export VAGRANT_NUM_NODES=1
-    make cluster-up
+export KUBEVIRT_NUM_NODES=1
+make cluster-up
 ```
 
 You could also run some build steps individually:
 
 ```bash
-    # To build all binaries
-    make
+# To build all binaries
+make
 
-    # Or to build just one binary
-    make build WHAT=cmd/virt-controller
+# Or to build just one binary
+make build WHAT=cmd/virt-controller
 
-    # To build all docker images
-    make docker
+# To build all docker images
+make docker
+```
+
+### Accessing the containerized nodes via ssh
+
+The containerized nodes are named starting from `node01`, `node02`, and so
+forth. `node01` is always the master of the cluster.
+
+Every node can be accessed via its name:
+
+```bash
+cluster/cli.sh ssh node01
```

Comment:
yay, thanks for this :) I was using cli directly and it was a giant string i'd have to copy out of my notes every time i wanted to ssh.